### PR TITLE
Feature/151560045/view rating and game stats

### PR DESCRIPTION
**Pull request by dinobi**
#### What does this PR do?
Adds a dashboard UI to the app
#### Description of Task to be completed?
After a user signup or signin action, a dashboard containing their game stats, their rating based on donations made on the platform will immediately be available, including an appropriate button to start a new game.
#### How should this be manually tested?
Do a signin or signup at http://rainer-cfh-staging.herokuapp.com
#### Any background context you want to provide?
Enable users track progress and contribution to the platform
#### What are the relevant pivotal tracker stories?
Story ID: #151560045
Story Type: Feature
#### Screenshots (if appropriate)

dashboard

#### Questions:

### CODE REVIEW

File: public/views/index.html [hunk 1]
```diff
@@ -67,22 +67,10 @@ <h1 class="red-text"><i class="fa fa-warning"></i></h1>
               <p>This game contains offensive, profane, explicit content created by <a href="#" >Cards Against 
                 Humanity</a> and provided here, uncensored and complete, in digital form. Human Resources made us 
                 say that.
+                {{dinoMessage}}
```
Comment (nebanat) [inline public/views/index.html]: Please use a better variable name other than dinoMessage

**Review by nebanat — APPROVED** [review]

File: public/views/index.html (hunk 1 above)
Comment (dinobi) [reply public/views/index.html]: This probably came from a pull request I made on Sat. Note: I worked only on a dashboard UI, and a dashboard controller.

Comment (ibonly) [reply public/views/index.html]: This is in your PR so you need to update it.

File: public/views/dashboard.html [hunk 2]
```diff
@@ -1,4 +1,196 @@
...
+          <p class="giving-quote white-text center">"If you don't give back, no one will like you&trade;" -
+            <a class="red-text" target="_blank" href="https://crowdrise.com"> crowdrise.com </a>
+          </p>
+        </div>
+      </section>
+      <!-- The team -->
+      <footer class="cfh-section section scrollspy" id="partners">
+          <nav class="container">
+            <div class="left">
+                <a href="#home" class="cfh-logo black-text"><img src="img/cfh-logo.png" alt="cfh-logo"/></a>
```
Comment (ibonly) [inline public/views/dashboard.html]: Why is the image path different from other implementation, check line 175?

File: public/views/index.html (hunk 1 above)
Comment (dinobi) [reply public/views/index.html]: I removed that line as soon as I saw it

Comment (ibonly) [reply public/views/index.html]: If you remove it, why is it still here? Kindly cross check.

File: public/views/dashboard.html (hunk 2 above)
Comment (dinobi) [reply public/views/dashboard.html]: That was an oversight. Resolved!

**Review by enodi — APPROVED** [review]

**Comment by ibonly** [comment]
Why was this merged without my approval?

**Pull request merged**